pg_lakehouse/README.md

```diff
@@ -0,0 +1,299 @@
+<h1 align="center">
+  <img src="../docs/logo/pg_lakehouse.svg" alt="pg_lakehouse" width="500px">
+<br>
+</h1>
+
+## Overview
+
+`pg_lakehouse` is an extension that transforms Postgres into an analytical query engine over object stores like S3 and table formats like Apache Iceberg. Queries are pushed down to [Apache DataFusion](https://github.com/apache/datafusion), which delivers excellent analytical performance. Combinations of the following object stores, table formats, and file formats are supported.
+
+### Object Stores
+
+- [x] Amazon S3
+- [x] S3-compatible object stores (e.g. MinIO)
+- [x] Local file system
+- [ ] Azure Blob Storage (coming soon)
+- [ ] Google Cloud Storage (coming soon)
+
+...and potentially any service supported by [Apache OpenDAL](https://opendal.apache.org/docs/category/services). See the Development section for instructions on how to [add a service](#adding-a-service).
+
+### File Formats
+
+- [x] Parquet
+- [x] CSV
+- [x] JSON
+- [x] Avro
+- [ ] ORC (coming soon)
+
+### Table Formats
+
+- [x] Delta Lake
+- [ ] Apache Iceberg (coming soon)
+
+`pg_lakehouse` is supported on Postgres 14, 15, and 16. Support for Postgres 12 and 13 is coming soon.
+
+## Motivation
```

Comment:
I think Motivation should be either first, or right below Overview but before the before the supported systems, no? Likely the first thing a new reader wants to read

Reply:
It's one sentence...